src/main/kotlin/com/pubnub/api/subscribe/eventengine/effect/HandshakeProvider.kt

```diff
@@ -4,5 +4,5 @@ import com.pubnub.api.endpoints.remoteaction.RemoteAction
 import com.pubnub.api.subscribe.eventengine.event.SubscriptionCursor
 
 fun interface HandshakeProvider {
-    fun handshake(channels: List<String>, channelGroups: List<String>): RemoteAction<SubscriptionCursor>
+    fun getRemoteActionForHandshake(channels: List<String>, channelGroups: List<String>): RemoteAction<SubscriptionCursor>
```

Comment:
`handshakeRemoteAction`?

Reply:
Ok, I will change it to getHandshakeRemoteAction